pkg/model/api/v1/variable/list.go

```diff
@@ -75,6 +75,7 @@ type ListSpec struct {
 	// CapturingRegexp is the regexp used to catch and filter the result of the query.
 	// If empty, then nothing is filtered. That's the equivalent of setting CapturingRegexp with (.*)
 	CapturingRegexp string        `json:"capturingRegexp,omitempty" yaml:"capturingRegexp,omitempty"`
+	Sort            string        `json:"sort,omitempty" yaml:"sort,omitempty"`
```

Comment:
that's not a random string here. It's an enum and it should be reflected in the backend

Reply:
it would be cool to have a description of this field even if it's obvious. You could describe the logic, what's happening when the value is not providing for example